bot.py

```diff
@@ -210,6 +209,8 @@ def _load_karma_from_json_file(self):
         karma_file = Path(self.karma_file_path)
         if not karma_file.is_file():
             print('No existing file found. Will start fresh.')
+            with open(self.karma_file_path, 'w') as file_ptr:
+                file_ptr.write('[]')
```

Comment:
Do we want to write `[]` into the file or do we just want to effectively "touch" the file to make it exist and let the format of the file be handled by the Karma events?

Reply:
We need the empty array, otherwise it's not parsable as JSON.

Reply:
This is at boot, right?  What information do you suggest we insert here?  AFAIK there won't be any new karma items (yet) to record.